### PR TITLE
regEXP to validate link URL

### PR DESCRIPTION
**Pull request by Gabriel903**
validates Link URL - RelatedArticle
fix: tests to pass
fix: travis ci to validate on node versions 7 and 10

### CODE REVIEW

File: src/RelatedArticle.js [hunk 1]
```diff
@@ -48,7 +66,9 @@ export default class RelatedArticle extends Component {
             placeholder={__("Link")}
             value={this.props.item.link}
             styles={{padding: "small"}}
-            onChange={this._handleLinkChange} />
+            onChange={this._handleLinkChange} 
+            error= {error && <p style={errorStyle}>{"Link Inválido"}</p>}
```
Comment (vierno) [inline src/RelatedArticle.js]: Could you please make this string i18n compatible?

File: tests/RelatedArticle_test.js [hunk 2]
```diff
@@ -65,10 +65,6 @@ describe("RelatedArticle", function() {
   it("updates entity on link change", function () {
     this.link.value = "new link";
     ReactTestUtils.Simulate.change(this.link);
-    expect(this.updateArticle.calledWith(
-      this.data.key,
-      "link",
-      "new link"
-    )).to.be.true;
+    expect(this.updateArticle.calledWith(this.data.key,"link","new link")).to.be.true;
```
Comment (vierno) [inline tests/RelatedArticle_test.js]: Why this change was needed?

File: src/RelatedArticle.js (hunk 1 above)
Comment (Gabriel903) [reply src/RelatedArticle.js]: No problem!

File: tests/RelatedArticle_test.js (hunk 2 above)
Comment (Gabriel903) [reply tests/RelatedArticle_test.js]: It was a mistake, undoing.

**Comment by vierno** [comment]
lgtm
